extensions/wikia/FandomComMigration/FandomComMigration.i18n.php

```diff
@@ -15,11 +15,13 @@
 $messages['de'] = array(
 	'fandom-com-migration-after' => 'Achtung: Wir haben die Domain deiner Community auf fandom.com umgezogen. Mehr dazu findest du auf [http://de.community.wikia.com/wiki/Hilfe:Domain-Umzug_auf_Fandom dieser Hilfeseite].',
 	'fandom-com-migration-before' => 'Achtung: Im Laufe der nächsten Wochen ziehen wir die Domain deiner Community um auf fandom.com. Mehr dazu auf [http://de.community.wikia.com/wiki/Hilfe:Domain-Umzug_auf_Fandom dieser Hilfeseite].',
+	'fandom-ucp-migration-before' => 'Achtung: Innerhalb der nächsten Wochen wird diese Community mitsamt aller Inhalte auf eine neue Wiki-Software übertragen. Erfahre [https://community.fandom.com/de/wiki/Benutzer_Blog:SpacePucky/Erste_Wikis_ziehen_zur_neuen_Plattform_um in diesem Blog-Beitrag] mehr darüber.',
```

Comment:
We have duplicated `fandom-ucp-migration-before` key or github is showing something wired?